protos/envelope.proto

```diff
@@ -0,0 +1,38 @@
+// https://raw.githubusercontent.com/secure-systems-lab/dsse/9c813476bd36de70a5738c72e784f123ecea16af/envelope.proto
```

Comment:
should include apache license here for completeness

Reply:
Of course! I'll fix that. 
I was actually thinking of removing the file entirely, and only pull it during build time. WDYT?

Reply:
oh, you don't need the entire license IMO, just something like what we do in sigstore:

```
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```